### PR TITLE
feat: Supply EOL for Docker

### PR DESCRIPTION
**Pull request by AaronBuxbaum**
I have no good way of testing if this will look right in Splunk/Datadog, but it logs the same as before to my local console:
Screen Shot 2020-03-25 at 1 05 49 PM

### CODE REVIEW

File: createLogging.js [hunk 1]
```diff
@@ -120,7 +119,7 @@ module.exports = function createLogging({
     return desc;
   }
 
-  logger.get = (id) => container.get(id);
```
Comment (itajaja) [inline createLogging.js]: weird linting changes that CI didn't complain about? 🤔

**Review by itajaja — COMMENTED** [review]
looks good but the formatting issue is bizarre, especially because CI didn't fail... maybe bump packages and dedupe?

**Comment by taion** [comment]
there's no tests in CI here; just the semantic PR check

you need to update deps and re-run prettier

**Review by itajaja — APPROVED** [review]

**Review by taion — APPROVED** [review]
worth a try

**Comment by taion** [comment]
wait, @AaronBuxbaum , none of those are multi-line logs? what do multi-line logs look like now?

**Comment by AaronBuxbaum** [comment]
> wait, @AaronBuxbaum , none of those are multi-line logs? what do multi-line logs look like now?

Datadog parses each line as a separate log line. If this works, or if we log in JSON, it collects them together into a single longer log. I’m a little skeptical here, plus I’m not sure key indexing will work (which is really valuable with Splunk), but we’ll see

**Comment by taion** [comment]
i mean literally your screenshot: 
image

does not show this functionality

**Comment by AaronBuxbaum** [comment]
> i mean literally your screenshot:
> image
> 
> does not show this functionality

Oh, yeah, that's my local console. It looks great there of course.

Here's datadog:

Screen Shot 2020-03-27 at 11 16 37 AM

**Comment by taion** [comment]
wait, so that's still getting broken up on datadog?

**Comment by AaronBuxbaum** [comment]
> wait, so that's still getting broken up on datadog?

That's how it is right now (pre-this PR). I'm not sure how to best test this:
1. running Datadog on local
2. setting up patch-package in lobot and patching 4c-logging
3. pushing something into 4c-logging and then pulling it into lobot

I was going for #3, but that's obviously very trial-and-error

**Comment by AaronBuxbaum** [comment]
> wait, so that's still getting broken up on datadog?

I tried this locally and it didn't fix anything the issue :'(

**Pull request closed without merging**